install-ci.yml

```diff
@@ -251,6 +251,20 @@
             socket: localhost:3310
             lazy-apps: true
 
+    - role: logging-proxy
+      logging_proxy_host: 127.0.0.1
+
+    - role: logrotate
+      logrotate_configs:
+        - name: zuul
```

Comment:
This name should probably be logging-proxy. I think this might overwrite the zuul config if both are included on the same host.